dashboards/grafana-dashboard-insights-rbac-operations.configmap.yaml

```diff
@@ -3807,7 +3784,7 @@ data:
             "current": {
               "selected": false,
               "text": "crcp01ue1-prometheus",
-              "value": "crcp01ue1-prometheus"
+              "value": "PC1EAC84DCBBF0697"
```

Comment:
Curious, what is this value?

Reply:
the reason is same as I described in the second comment